src/mudlet.cpp

```diff
@@ -2375,6 +2375,14 @@ void mudlet::setEditorTextoptions( const bool isTabsAndSpacesToBeShown, const bo
     emit signal_editorTextOptionsChanged( mEditorTextOptions );
 }
 
+void mudlet::setEchoLuaErrors( const bool isEchoLuaErrorsEnabled )
```

Comment:
Yeah, the calls _up to_ the `mudlet` class are generally for application wide settings IIRC - just set the `pHost->mEchoLuaErrors` in the place suggested.